lms/services/__init__.py

```diff
@@ -20,6 +20,9 @@
 
 def includeme(config):
     config.register_service_factory("lms.services.http.factory", name="http")
+    config.register_service_factory(
+        "lms.services.blackboard_api.factory", name="blackboard_api_client"
+    )
```

Comment:
If we are registering a new service, this seems like a good time to use the normal interface style.
```suggestion
    config.register_service_factory(
        "lms.services.blackboard_api.factory", iface=BlackboardAPIClient
    )
```